src/lib/components/DeckGLMap/Map.tsx

```diff
@@ -82,8 +83,15 @@ const Map: React.FC<MapProps> = ({
                     getCursor={({ isDragging }): string =>
                         isDragging ? "grabbing" : "default"
                     }
-                    getTooltip={(info: PickInfo<unknown>): string | null => {
-                        return (info.object as Feature)?.properties?.name;
+                    getTooltip={(
+                        info: PickInfo<unknown> | WellsPickInfo
+                    ): string | null | undefined => {
+                        const well_info = info as WellsPickInfo;
+                        if (well_info) {
+                            return well_info.logName;
```

Comment:
`info as WellPickInfo` will not return `null` if `info` is not a `WellPickInfo`. I think it should be something like this:
```suggestion
                        if ((info as WellsPickInfo)?.logName) {
                            return (info as WellsPickInfo).logName;
```

Reply:
Done